flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/PlannerCallProcedureOperation.java

```diff
@@ -138,6 +137,14 @@ private Object[] getConvertedArgumentValues(
         return argumentVal;
     }
 
+    private ProcedureContext getProcedureContext(TableConfig tableConfig) {
+        Configuration configuration = (Configuration) tableConfig.getRootConfiguration();
```

Comment:
nit:
Configuration configuration = new Configuration(tableConfig.getRootConfiguration());

Reply:
Yeh ,it's better, but the Configuration class doesn't provide a construct as Configuration(ReadableConfig config), it has to be : Configuration configuration = new Configuration((Configuration)tableConfig.getRootConfiguration());

Reply:
Sorry for the mistake. Yes, it should be 
Configuration configuration = new Configuration((Configuration)tableConfig.getRootConfiguration());